hera_qm/tests/test_ant_metrics.py

```diff
@@ -222,28 +139,6 @@ def test_antpol_metric_sum_ratio():
     assert crossPolRatio == {(0, 'x'): .5, (0, 'y'): .5}
 
 
-def test_average_abs_metrics():
```

Comment:
Sure. This test was removed because we used to average together the redcorr metrics and the meanVij metrics to get our "final metric" we no longer do that, so we don't need this method for averaging two metrics together. We can always add it back if we devise a new metric that we want to average with meanVij, but I think we're moving away from relying on antmetrics and more toward observers and dashboards.

Reply:
Thanks!